src/chrome/content/rules/BitMEX.com.xml

```diff
@@ -1,4 +1,10 @@
-<ruleset name="BitMEX.com">
+
+<!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://bitmex.com/ => https://bitmex.com/: Redirect for 'https://bitmex.com/' missing Location
+
```

Comment:
The script appears to be reporting that there is a missing location header somewhere along the redirect chain for either the HTTP or HTTPS version, but I can't reproduce this manually. I'll re-enable this rule and debug. https://github.com/hiviah/https-everywhere-checker/issues/9